core/src/test/scala/unit/kafka/integration/KafkaServerTestHarness.scala

```diff
@@ -88,7 +88,8 @@ abstract class KafkaServerTestHarness extends ZooKeeperTestHarness {
     // default implementation is a no-op, it is overridden by subclasses if required
     configureSecurityBeforeServersStart()
 
-    servers = configs.map(TestUtils.createServer(_)).toBuffer
+    for (config <- configs)
```

Comment:
Worth adding a comment so that people don't change the code back to what it was.